infra/charts/feast-feature-server/templates/instrumentation.yml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.metrics }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: feast-instrumentation
+  namespace: {{ .Release.Namespace }}
+spec:
+  exporter:
+    endpoint: http://{{ .Values.service.name }}-collector.{{ .Release.Namespace }}.svc.cluster.local:4318 # This is the default port for the OpenTelemetry Collector
```

Comment:
The `svc.cluster.local` part could be omittes, IIUC (in both places, also previous file)
Also, the port numbers, shouldn't they be configurable with a default in the values file, instead?